src/django_components/apps.py

```diff
@@ -66,6 +67,7 @@ def ready(self) -> None:
 
         # Register the dynamic component under the name as given in settings
         registry.register(app_settings.DYNAMIC_COMPONENT_NAME, DynamicComponent)
+        registry.register("error_fallback", ErrorFallback)
```

Comment:
I didn't add a setting to change the name under which the ErrorFallback is accessed from within the templates. We can add it later if need be.